### PR TITLE
added note() and n2v() ASL library functions

### PR DESCRIPTION
**Pull request by isaacpearl**
Added note() and helper n2v() (note to voltage) functions to asllib, which should allow for easier/more readable melodic sequencing in ASL.

### CODE REVIEW

File: lua/asllib.lua [hunk 1]
```diff
@@ -11,6 +11,16 @@ function negate( v )
...
 end
 
+function n2v( n )
+	if type(n) == 'function' then
+		return function () return n()/12 end
+	else return n/12 end
+end
+
+function note( noteNum, duration )
+	return{to(n2v(noteNum), 0), to('here', duration)}
```
Comment (trentgill) [inline lua/asllib.lua]: could you update the layout here to make it match the rest of asllib calls. something like:
```
return{ to( n2v(noteNum), 0 )
      , to( 'here', duration )
      }
```
i think it makes it clearer that it's a chaining of 2 `to`s.

**Comment by isaacpearl** [comment]
done!

**Comment by isaacpearl** [comment]
this is looking weird in the source code on here (vs my editor settings) - working on it still

**Comment by isaacpearl** [comment]
ok! this is actually done now.

**Pull request merged**